data/wanderer dire storm.txt

```diff
@@ -0,0 +1,136 @@
+
+ship "Dire Storm"
+	sprite "ship/dire storm"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 40500000
+		"shields" 54128
+		"hull" 48962
+		"required crew" 72
+		"bunks" 237
+		"mass" 832
+		"drag" 13.9
+		"heat dissipation" 0.5
+		"fuel capacity" 816
+		"cargo space" 424
```

Comment:
That's a huge amount of cargo space. Is this a cargo ship?

Reply:
Spam.

Reply:
...What?